test/common/verify_3scale_UIBBT_alerts.go

```diff
@@ -0,0 +1,187 @@
+package common
+
+import (
+	"context"
+	goctx "context"
+	"fmt"
+	openshiftappsv1 "github.com/openshift/api/apps/v1"
+	k8sappsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/json"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/retry"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+type Alert struct {
+	Labels struct {
+		AlertName string `json:"alertname"`
+	} `json:"labels"`
+	State string `json:"state"`
+}
+
+type AlertsResults struct {
+	Alerts []Alert `json:"alerts"`
+}
+
+func TestThreeScaleUIBBTAlerts(t TestingTB, ctx *TestingContext) {
+
+	// Get the kube client and namespace from the shared test context
+	client := ctx.Client
+
+	prefix := GetNamespacePrefix()
+	if prefix == "" {
+		t.Fatalf("Failed to get namespace prefix")
+	}
+
+	// Scale down 3scale-operator to 0 pods
+	err := scaleDeployments(context.TODO(), t, client, prefix+"3scale-operator", "threescale-operator-controller-manager-v2", 0)
+	if err != nil {
+		t.Fatalf("Failed to scale down 3scale-operator: %v", err)
+	}
+
+	time.Sleep(1 * time.Minute)
+
+	// Scale down system-app to 0 pods
+	err = scaleDeploymentsConfig(t, "system-app", prefix+"3scale", 0, client)
+	if err != nil {
+		t.Logf("Failed to scale down 3scale-operator: %v", err)
```

Comment:
Yeah I will fix that string, sorry

I don't use `t.Fatalf`, because if I would use it the test would fail and deployments which were scaled down would not be scalled back, because function which is responsible for scaling thing back is further in the code and is also "defered". That's my understanding now

Reply:
So I did more research on this and for now it might be better option to use `t.Errorf` instead of `t.Fatalf` or `t.Log`? Becuase that would cause test to fail, but it will not stops it's execution, so scaling back to default state will happen. What do you think?

Reply:
I think that using `t.Fatalf` is more appropriate because there is no point in continuing running the test if there was an error scaling the 3scale operator down. The test would just take some time and fail at the end anyway.